data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+			choice
+				`	"Well that's all well and good. But I have my own business to attend to now. Good luck, and goodbye."`
+					decline
+				`	(Tell her about the Coalition.)`
+			`	You are about to reveal sensitive information that you might not want to spread. You are on a busy street with high foot traffic. You tell Anna you have something to tell her, and ask her to walk to a secluded corner of the store.`
+			`	Once confirming that you and Anna will not be heard, you begin explaining the Coalition to Anna, going in-depth about their history and their culture, and their long history of opposing the Quarg, seeing as Anna has less than positive opinions of them.`
+			`	"Will they really like my books?"`
+			choice
+				`	"They will, hopefully."`
+				`	"Well, it's better to try than to just give up."`
+			`	"I hope they do. Is there anything more you'd like to tell me about the Coalition?"`
+			choice
+				`	"By any chance, are you afraid of large beetles and spiders?"`
+			`	"No, not at all. I'm not afraid of them. Why do you ask?"`
+			choice
+				`	"Because the Coalition are a union of three alien species, who look like spiders, beetles, and centaurs.`
+			`	"Interesting. Their stories might just inspire my next novel. But, they wouldn't speak our language. How are they going to read my books?"`
+			choice
+				`	"They have a translation device."`
+				`	"I'm sure they can learn the language."`
+			`	"That's reassuring... but, how am I even going to get there?"`
+			choice
+				`	"I'll take you there."`
+			`	"Do I have to pay anything? As a struggling author, I can't afford off-world transport at all, let alone to a completely different alien race!"`
+			choice
+				`	"It's okay. I'm willing to make an exception for you."`
+			`	"Thank you so much! Just wait for a while, I need to finish my book signing event. It should be roughly 45 minutes before it's over, and then we can go to whatever this Coalition thing is," says Anna.`
+			`	You reply, "Good luck."`
+			`	As you wait for Anna, you look over your maps of Coalition space to determine the most suitable place for Anna. After much deliberation and thought, you decide on <destination>.`
+			`	After approximately an hour of waiting, Anna appears outside of your ship, knocking on the hatch. You open the hatch to let her inside, as you lead her to her bunk and inform her of the destination. You prepare to leave the spaceport.`
+				accept
+	on complete
+		conversation
+			`	You leave the ship with Anna following you. The moment you are out of your ship, three Kimek surround you and Anna. The Kimek in the middle speaks, and then brief silence. The silence is broken as their translation box speaks.`
```

Comment:
If you want them to approach you immediately, I'd make it clear it's Heliarch agents who approach you and Anna here. If you want just regular civilian Kimek, have it take a while for them to notice you've brought another human. Just a bit of a difference as the Heliarchs'd know earlier that there's another human aboard (we're not taking regular crew into account for this as those aren't acknowledged).